tests/setup.js

```diff
@@ -3,24 +3,20 @@
  * @author  TheJaredWilcurt
  */
 
-const os = require('os');
-const processPlatform = process.platform;
-const testHelpers = require('@@/testHelpers.js');
+import process from 'node:process';
 
-if (os.platform() !== 'win32') {
+import testHelpers from './testHelpers.js';
+
+if (process.platform !== 'win32') {
```

Comment:
I don't remember why, but I do remember setting up all the process.platform/os.platform stuff was done intentionally and was annoying to get working. Not sure why, but I think it was more of a cross-platform node thing, than a jest thing. so would probably still be needed, but not sure.